src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Adaptoinator.java

```diff
@@ -4,14 +4,15 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class Adaptoinator extends DrawPanelController implements Job2dDriver {
```

Comment:
Interesting name, but it doesn't explain the purpose of the adapter or what was actually adapted

Inheritance is not used (no super drawLine calls). Should be removed!